playwright-tests/createNft/actions.ts

```diff
@@ -0,0 +1,23 @@
+import { Page } from '@playwright/test';
+import { DataTestIdsEnum } from '../../src/localConstants/dataTestIds.enum';
+import { GlobalDataEnum } from '../utils/enums';
+
+interface NftCreation {
```

Comment:
This should have the suffix `Type` in order to distinguish from other components or helpers.